package.json

```diff
@@ -28,9 +28,6 @@
     "index.d.ts",
     "index.js"
   ],
-  "dependencies": {
-    "builtins": "^5.0.0"
-  },
```

Comment:
Can you keep this as `"dependencies": {},`?

Reply:
Sure. It might be the case that a package manager removes this the next time the dependencies are modified via the CLI, though.